ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/constraint/model/ConstraintAnnotation.java

```diff
@@ -1,26 +1,26 @@
 /*
- * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
  *
- * WSO2 LLC. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
  */
-package io.ballerina.openapi.converter.service;
+package io.ballerina.openapi.service.mapper.constraint.model;
```

Comment:
Do we really need to create another package to store this model. AFAIK we only use packages when there are multiple components to group together. IMO this class could reside in the `constraint` package without any issue as there are no more model classes in the specified package.

Reply:
I used a separate package to organize the code properly. May be in the future there can be more than one model for constraint mapping.

Reply:
Yeah, but until then we don't need to have a separate package right. Usually there are two ways to organize java classes into packages:
- group the classes into one package for a specific functionality (login, signup etc)
- group the classes into one package which does a similar thing (repository, service etc)

Here we are using the first approach. And unless we have multiple model classes we do not need to have a separate package inside `constraint` package.

Reply:
Fixed via [379f404](https://github.com/ballerina-platform/openapi-tools/pull/1564/commits/379f404de0103dfa26c6a4e1ed52c15944a61eb5)